bin/GameIndex.dbf

```diff
@@ -1264,7 +1270,8 @@ Serial = SCES-50354
 Name   = Klonoa 2 - Lunatea's Veil
 Region = PAL-M5
 Compat = 5
-eeClampMode = 3		//Objects needed appear in wrong places without it
+eeClampMode = 3 // Objects needed appear in wrong places without it 
```

Comment:
> Objects needed appear in wrong places without it 

Seems like someone added 'needed' to this line by mistake. It makes sense without it.